frontend/src/components/PLAYGROUND/TableWithModal/index.tsx

```diff
@@ -34,9 +39,10 @@ export const TableWithModal = ({
   ) : (
     <>
       <Table
-        rows={tableData.data.data.data}
+        rows={tableData.data ? tableData.data.data.data : []}
```

Comment:
but why is there something named `tableData.data.data.data `?

Reply:
It's to do with the fact the every library we use for data such as fastify, knex and react query each have a data property.